fastlane/.gitignore

```diff
@@ -0,0 +1 @@
+installer/
```

Comment:
You're introducing a gitignore here, whereas we are adding stuff like `fastlane/test_output` for the .gitignore at the root level. Can we stick with one to have everything in one place?